client/my-sites/plugins/plugin-management-v2/plugin-action-status/style.scss

```diff
@@ -0,0 +1,37 @@
+@import '@wordpress/base-styles/_breakpoints.scss';
+@import '@wordpress/base-styles/_mixins.scss';
```

Comment:
Can we replace these two with the following:

```
@import '@wordpress/base-styles/breakpoints';
@import '@wordpress/base-styles/mixins';
```
The CSS linter complains about the existing imports:

```
  1:1   ✖  Unexpected leading underscore in imported partial name            scss/at-import-no-partial-leading-underscore
  1:46  ✖  Unexpected extension ".scss" in @import                           scss/at-import-partial-extension  
```